sqlmesh/core/config/connection.py

```diff
@@ -42,6 +42,8 @@ class ConnectionConfig(abc.ABC, BaseConfig):
     concurrent_tasks: int
     register_comments: bool
 
+    type_: str = Field(alias="type", default="none")
```

Comment:
Pydantic V2 `dict()` does not include `type` in for child instances unless `type` is also defined in the base class.

Reply:
Adding this as a code comment might be helpful